docs/design/autoware-interfaces/ad-api/types/autoware_ad_api_msgs/msg/driving_state.md

```diff
@@ -0,0 +1,16 @@
+# autoware_ad_api_msgs/msg/DrivingState
```

Comment:
Why do you not link to the msg file in the package repository?
I'm concerned about it would be double maintenance.

Reply:
This page can provide links to "this type uses" and "this type is used by" for convenience. Maintenance is not difficult as it is generated by a script from the message package.